src/main/java/seedu/address/storage/JsonAdaptedPassenger.java

```diff
@@ -65,9 +70,10 @@ public JsonAdaptedPassenger(Passenger source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
         address = source.getAddress().value;
-        tripDay = source.getTripDay().value;
-        tripTime = source.getTripTime().value;
-        driver = source.getDriverStr();
+        tripDayStr = source.getTripDay().toString();
+        tripTimeStr = source.getTripTime().toString();
```

Comment:
perhaps we should call TripDay and TripTime's source.getTripDayAsString() method so as to not violate the law of demeter?

Reply:
Thanks for the comment! I will make the relevant changes.